src/App.js

```diff
@@ -1,19 +1,17 @@
-import React from "react";
 import { hot } from "react-hot-loader";
+import React from "react";
```

Comment:
At a glance it looks like this can be deleted now.

Reply:
the hot loader?